opencog/reasoning/RuleEngine/rules/pln/evaluation-to-member.scm

```diff
@@ -0,0 +1,64 @@
+; ============================================================================= 
+; GeneralEvaluationToMemberRule
+;	EvaluationLink (pred D (ListLink B C))
+;		becomes
+;	MemberLink( B (SatisfyingSetLink( Variable $X 
+;		(EvaluationLink (pred D (ListLink X C))))))
+; -----------------------------------------------------------------------------
+
+(define pln-rule-evaluation-to-member
+	(BindLink
+		(ListLink
+			(VariableNode "$B")
+			(VariableNode "$C")
+			(VariableNode "$D")
+			(TypedVariableLink
+				(VariableNode "$D")
+				(VariableTypeNode "PredicateNode")))
+		(ImplicationLink
+			(EvaluationLink
+				(VariableNode "$D")
```

Comment:
The current structure of this code does not account for undefined number of arguments. So, to workaround that problem, I am planning to use the following approach:
For evaluation-to-member:-
1. Extract all Evaluation Links in atomspace using (cog-get-atoms 'EvaluationLink).
2. For each Link in the list created above, Create a Member Link manually using 'car' and 'cdr'.
3. Set TVs of the New Member Link same as that of the Evaluation Link from which it was created.

I started implementing this but haven't yet completed it. I'll wrap it up tomorrow. Does this approach seem good or is there a cleaner/better way to do this?